Checkbox/Checkbox.module.less

```diff
@@ -4,32 +4,63 @@
 @import "../styles/skin.less";
 
 .checkbox {
+	display: inline-block;
+	position: relative;
+
 	.applySkins({
 		@size: @agate-checkbox-size;
 
 		width: @size;
 		height: @size;
-		border: @agate-checkbox-border-width solid @agate-checkbox-border-color;
 		border-radius: @agate-checkbox-border-radius;
 		color: @agate-checkbox-text-color;
 		background-color: @agate-checkbox-bg-color;
 
+		.bg {
+			position: absolute;
+			width: @agate-checkbox-bg-height-width;
+			height: @agate-checkbox-bg-height-width;
+			border-radius: 50%;
+			.position(@agate-checkbox-container-position);
+			z-index: -1;
```

Comment:
Why do we need this?

Reply:
This was left by mistake, I will delete it